app/helpers/value_helper.rb

```diff
@@ -69,9 +69,14 @@ def links_to_collections(args)
     end.join('<br>').html_safe
   end
 
+  # Discussed removing this method because the field should not be modified.
   def value_for_originInfo_date_created_tesim(args)
-    val = Time.parse(args[:document][args[:field]].first)
+    field_value = args[:document][args[:field]].first
+    # Try to parse the string as a time value
+    val = Time.parse(field_value)
```

Comment:
It looks like argo does not explicitly set the app time zone, so `Time.zone` is the default UTC.  Can we assume that this data should be parsed as a UTC value?